src/HornedBeasts.js

```diff
@@ -1,15 +1,45 @@
 import React, { Component } from 'react'
+import Card from 'react-bootstrap/Card'
 
 class HornedBeast extends Component {
-  render() {
-    return (
-        <div>
-            <h2>{this.props.title}</h2>
-            <img src={this.props.src} alt={'photo of ' + this.props.title} title={this.props.title + 'photo'}/>
-            <p>{this.props.description}</p>
-        </div>
-    )
-  }
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            likes: 0, 
+            selected: false
+        }
+    }
+
+    handleClick = () => {
+        this.setState({
+            likes: this.state.likes + 1
+        });
+        if(!this.selected){
+            this.setState({
+                selected: true
+            });
+        }
+    }
+
+    render() {
+        return (
+            <Card bg='dark'>
+                    <Card.Header>{!this.state.selected ? this.props.title : 'Good Choice!'}</Card.Header>
+                <Card.Img variant="top" onClick={this.handleClick} src={this.props.src} alt={'photo of ' + this.props.title} title={this.props.title + 'photo'} />

```

Comment:
Hi there!

In lab submission you asked how to get cards to be all same height: If you inspect the image, you will see `card-img-top`. This has styles applied to it. If you change them to be:

```
width: 100%;
    height: 15vw;
    object-fit: cover;
```
All will be same height :)